helm-chart/templates/cvat_kvrocks/deployment.yml

```diff
@@ -0,0 +1,90 @@
+{{- if .Values.cvat.kvrocks.enabled }}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Wouldn't a StatefulSet make more sense here? I assume every replica of Kvrocks would need its own storage volume; that matches StatefulSet semantics better.